### DISCUSSION TITLE
Support for Enocean LAN Gateway

### DISCUSSION THREAD

**Discussion by TTr0n** [mak-gitdev/HA_enoceanmqtt#137] [Request]
Hi, is it possible using EnoceanMQTT with a Lan gateway ( TCM515 ) ?
TCP Connection with port exists and is working. 

Greetings

**Comment by Seqway** [comment]
I have same scenario and needed to implement the Lan GW. Would be nice to get a feedback

**Comment by tomknocke** [comment]
Did somebody found an answer? Iam also searching for TCP LAN option.
Homeassistant is running in a VM in my Network, it like to get rid of USB passtrough.
Thanks
